ReText/__init__.py

```diff
@@ -98,6 +98,7 @@ def getBundledIcon(iconName):
 	'useWebEngine': False,
 	'useWebKit': False,
 	'windowGeometry': QByteArray(),
+	'paperSize': 'A4',
```

Comment:
Why default explicitly to `A4`? Maybe leave the default empty and use Qt’s defaults?

Reply:
Done